types/video-react/index.d.ts

```diff
@@ -0,0 +1,510 @@
+// Type definitions for video-react 0.15.0
+// Project: https://github.com/video-react/video-react
+// Definitions by: Fabio Nettis <https://github.com/fabio-nettis>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import { LegacyRef } from 'react';
+
+declare module 'video-react' {
```

Comment:
The contents of a module should be put directly in the file instead of being wrapped in `declare module`